rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/dto/RestMapPage.java

```diff
@@ -0,0 +1,47 @@
+package org.ow2.proactive_grid_cloud_portal.scheduler.dto;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.xml.bind.annotation.XmlRootElement;
+
+
+@XmlRootElement
+public class RestMapPage<K extends Serializable, V extends Serializable> {
+
+    private int size;
+
+    private HashMap<K,V> map;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The type of the "map" object should be an interface such as "Map" rather than the implementation "HashMap". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1319)